### ISSUE TITLE
does smtp send mail must be 25 port of targe host?

### ISSUE THREAD

**Issue by MrSong0607** [foxcpp/maddy#694]
# Use case

case most of vps has disable outbound traffice with port 25, like vultr , digital ocean

# Your idea for a solution

have some way to set the default port of smtp sent, like 465, 587, or other way, just like proxy the outbound request over socks?

**Comment by taoso** [comment]
Yes. 25 port is using for mail exchange. The 465 and 587 are for mail submission, which means they are only used between you own mail client and smtp server. You smtp will always use the 25 port to connect the out domain server.

**Comment by MrSong0607** [comment]
> Yes. 25 port is using for mail exchange. The 465 and 587 are for mail submission, which means they are only used between you own mail client and smtp server. You smtp will always use the 25 port to connect the out domain server.

thanks for reply, can i proxy the 25 port smtp request ?

**Comment by greatbody** [comment]
This question is exactly what I had a few minutes ago. Thanks a lot for answering.
